python/ray/includes/global_state_accessor.pxi

```diff
@@ -78,6 +78,12 @@ cdef class GlobalStateAccessor:
             return c_string(object_info.get().data(), object_info.get().size())
         return None
 
+    def get_all_heartbeat(self):
```

Comment:
Can you add docstring for this method? Also, please make it clear it is for "momental" heartbeat. I had some impression that `getAllHeartbeat` will return historical heartbeat information (could be just me though).

Reply:
sure.